draft-barnes-acme.md

```diff
@@ -133,9 +133,6 @@ Subject Public Key:
 Account Key Pair:
 : A key pair for which the ACME server considers the holder of the private key authorized to manage certificates for a given identifier.  The same key pair may be authorized for multiple identifiers.
 
-Recovery Token:
```

Comment:
I would suggest that as with contact-based recovery, we punt on this for the purposes of this PR, and re-add it shortly.  I have a proposal that is 80% baked that I will share in email shortly.